modules/ROOT/pages/testing-oauth-modules.adoc

```diff
@@ -0,0 +1,71 @@
+= Testing OAuth Enabled Connectors
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule runtime engine supports authorization through various OAuth grant types including `Authorization Code`. This grant type allows an external system, for example your Mule application, to operate on behalf of a user without the need to share the user's credentials. However, because the `Authorization Code` grant type is designed to force human intervention in the authentication process that identifies users using OAuth (OAuth dance), building an automated test for an application using this grant type is challenging.
+
+You can use the Storage Event processor to bypass the whole OAuth dance phase by manually providing a valid access token. It is your responsibility to manually obtain (and eventually update) such token.
+
+Consider the following configuration of the Salesforce connector using OAuth authentication and a user defined object store to hold the tokens:
+
+[source,xml,linenums]
+----
+<os:config name="TokenObjectStore_Config" />
+
+<os:object-store name="tokenStore" persistent="true" config-ref="TokenObjectStore_Config" />
+
+<sfdc:config name="sfdcConfig" ...>
+	<sfdc:oauth-connection ....>
+		<sfdc:oauth-authorization-code ... />
+<sfdc:oauth-callback-config ... />
+<sfdc:oauth-store-config objectStore="tokenStore" />
+	</sfdc:oauth-connection>
+</sfdc:config>
+----
+
+To use that token in an MUnit test, create a flow and insert your tokens in an MUnit Storage Event processor:
```

Comment:
Should we specify the syntax or name representing "that token"?
Why does the sentence later refer to more than one token: "your tokens"?
Is there a text that defines the types of tokens, such as what I see here as access token, refresh token? Are there other types? Do we have to explain them here?